src/main/scala/net/cakesolutions/internal/CakeDockerUtils.scala

```diff
@@ -112,7 +120,11 @@ private[cakesolutions] object CakeDockerUtils {
 
   private def containerIds(
     ymlFiles: Seq[File]
-  )(implicit log: Logger): Seq[String] = {
+  )(
+    implicit log: Logger,
+    projectRoot: String,
```

Comment:
I know this is "just" SBT, but I don't think it's generally a good idea to have implicit parameters for such common types. It's so easy to accidentally pick up something you don't want to end up in the implicit parameters.

Reply:
Understood wrt types. I'll change this `String` to be a `File` type. Not ideal, but that should remove the majority of worries here?

Reply:
Yeah, no problem. I doubt it's going to be frequent enough of a problem. It's going to be a hell of a thing to debug, if it does happen though.